### PR TITLE
Feat: FSUI `Checkbox` component.

### PR DESCRIPTION
**Pull request by eduardoformiga**
Depends on: 
https://github.com/vtex/faststore/pull/1504
https://github.com/vtex-sites/nextjs.store/pull/295

## What's the purpose of this pull request?

Aims to use the `Checkbox` components from the new architecture: `faststore/ui `-> `faststore/components`. 
It also removes the component from the storybook.

## How does it work?

Adapts the store to use the new component.

## How to test it?

The storybook and Pages should show the Checkboxes just as we can see previously.

## References
https://github.com/vtex/faststore/pull/1504
https://github.com/vtex-sites/nextjs.store/pull/295

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "@faststore/api": "^1.12.17",
     "@faststore/graphql-utils": "^1.11.8",
     "@faststore/sdk": "^1.11.8",
-    "@faststore/ui": "https://pkg.csb.dev/vtex/faststore/commit/5d1fdfe6/@faststore/ui",
+    "@faststore/ui": "https://pkg.csb.dev/vtex/faststore/commit/ad3dcb1f/@faststore/ui",
```
Comment (eduardoformiga) [inline package.json:36]: 🎗️ Update the version.

File: src/components/search/Filter/Facets.tsx [hunk 2]
```diff
@@ -1,12 +1,11 @@
-import { Label as UILabel, List as UIList } from '@faststore/ui'
+import { Checkbox, Label as UILabel, List as UIList } from '@faststore/ui'
```
Comment (hellofanny) [inline src/components/search/Filter/Facets.tsx:1]: I was wondering, should we keep the previously pattern, WDYT? When importing components from fs/ui use the alias with UI prefix. 🤔

Comment (hellofanny) [inline src/components/search/Filter/Facets.tsx:1]:
```suggestion
import { Checkbox as UICheckbox, Label as UILabel, List as UIList } from '@faststore/ui'
```

Comment (hellofanny) [reply src/components/search/Filter/Facets.tsx:1]: If yes, remember to update in the code below :)

**Review by hellofanny — APPROVED** [review]
Weeeell donee forms! 🥳

File: src/components/search/Filter/Facets.tsx (hunk 2 above)
Comment (eduardoformiga) [reply src/components/search/Filter/Facets.tsx:1]: Nice! I think it makes more sense when we have a custom checkbox created here in the starter. But I'll keep the pattern for the sake of don't make it confuse.

**Pull request merged**